webboot/.bb/webboot.go

```diff
@@ -0,0 +1,218 @@
+// Copyright 2019 the u-root Authors. All rights reserved
```

Comment:
this file should not be here. 
git reset HEAD^ webboot/.bb/webboot.go

Reply:
Yeah i don’t know what happened, it added a file called .bb, let me make some changes